### PR TITLE
PAPP-26544: new parameter in file reputation action

### PR DESCRIPTION
**Pull request by pdesai-crest**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [ ] Testing of all the changes has been performed (for bug fixes / features)
- [ ] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [ ] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [ ] Verify all checks are passing.
- [ ] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [ ] Do NOT submit updates to dependencies unless it fixes an issue.
#### Please check the type of change your PR introduces:
- [ ] New App
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Security Considerations (REQUIRED)
- [ ] If you are exposing any endpoints using a [REST handler](https://docs.splunk.com/Documentation/SOAR/current/DevelopApps/RESTHandlers), 
  please document them in the `readme.html`.
- [ ] If this is a new connector or you are adding new actions
    - [ ] Please document in the `readme.html` all methods (eg, OAuth) used to authenticate 
      with the service that the connector is integrating with.
    - [ ] If any actions are unable to run on SOAR Cloud, please document this in the `readme.html`.
- [ ] Are you introducing any new cryptography modules? If yes, please elaborate their purpose:
- [ ] Are you are accessing the file system? If yes, please verify that you are only accessing paths returned through 
the [Vault](https://docs.splunk.com/Documentation/SOARonprem/5.2.1/DevelopApps/AppDevAPIRef#Vault) API.
- [ ] Are you are marking code to be ignored by Semgrep with [`nosemgrep`](https://semgrep.dev/docs/ignoring-files-folders-code/#ignoring-code-through-nosemgrep)?
If yes, please provide justification in an additional comment next to the ignored code.

  

## Release Notes (REQUIRED)
- Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.

## What is the current behavior? (OPTIONAL)
- Describe the current behavior that you are modifying.

## What is the new behavior? (OPTIONAL)
- Describe the behavior or changes that are being added by this PR.

## Other information (OPTIONAL)
- Any other information that is important to this PR such as screenshots of how the component looks before and after the change.

## Pay close attention to (OPTIONAL)
- Any specific code change or test case points which must be addressed/reviewed at the time of GA release.

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

**Comment by pdesai-crest** [comment]
The integration test pipeline is failing due to timeout as it takes more than one hour. Attaching the pipeline link from pytest for the same: https://cd.splunkdev.com/phantom-apps/app-tests/-/pipelines/6009107

File: crowdstrikeoauthapi.json [hunk 1]
```diff
@@ -606,6 +606,11 @@
                         "i-058fca45cda978e00"
                     ]
                 },
+                {
+                    "data_path": "action_result.data.*.kernel_version",
+                    "data_type": "string",
+                    "example_value": "10.0.19044.1766"
```
Comment (ighelani-crest) [inline crowdstrikeoauthapi.json:612]: It should be List and key is "example_values" not "example_value".
```
"example_values": [
    "10.0.19044.1766"
]
```
Please check for all the occurrences.

File: crowdstrikeoauthapi_connector.py [hunk 2]
```diff
@@ -28,7 +29,7 @@
 import phantom.utils as util
 import pytz
 import requests
-import simplejson as json
+# import simplejson as json
```
Comment (ighelani-crest) [inline crowdstrikeoauthapi_connector.py:32]: We can remove unnecessary imports instead of commenting out.

**Review by ighelani-crest — CHANGES_REQUESTED** [review]

File: crowdstrikeoauthapi_connector.py (hunk 2 above)
Comment (pdesai-crest) [reply crowdstrikeoauthapi_connector.py]: Done

File: crowdstrikeoauthapi.json [hunk 3]
```diff
@@ -12,12 +12,18 @@
...
+        {
+            "name": "Katie Tanner"
         }
     ],
     "license": "Copyright (c) 2019-2022 Splunk Inc.",
-    "app_version": "3.6.1",
+    "app_version": "3.7.0",
     "latest_tested_versions": [
-        "Cloud, API api.crowdstrike.com, April 4 2022, API version: v1, v2"
+        "Cloud, API api.crowdstrike.com, July 8 2022, API version: v1, v2"
```
Comment (alexa-phantom) [inline crowdstrikeoauthapi.json:26]:
```suggestion
        "Cloud, API api.crowdstrike.com, July 8 2022, API versions: v1, v2"
```

Comment (pdesai-crest) [reply crowdstrikeoauthapi.json]: Done

**Review by alexa-phantom — APPROVED** [review]

**Review by rgil-splunk — APPROVED** [review]

**Review by ighelani-crest — APPROVED** [review]

**Pull request merged**